repo/settings.py

```diff
@@ -27,7 +27,7 @@
 # SECURITY WARNING: don't run with debug turned on in production!
 DEBUG = config("DEBUG", cast=bool, default=True)
 
-ALLOWED_HOSTS = []
+ALLOWED_HOSTS = ["localhost", "7848-105-113-35-95.eu.ngrok.io"]
```

Comment:
move allowed hosts to env variable, so we don't have personal stuffs showing here